core/vm/interpreter.go

```diff
@@ -74,6 +70,10 @@ func NewEVMInterpreter(evm *EVM) *EVMInterpreter {
 		table = &istanbulInstructionSet
 	case evm.chainRules.IsConstantinople:
 		table = &constantinopleInstructionSet
+	case evm.chainRules.IsPrague:
+		// TODO replace with prooper instruction set when fork is specified
+		// table = &shanghaiInstructionSet
+		table = &byzantiumInstructionSet
```

Comment:
This is another fix needed when the replay crosses the border between Byzantium and Constantinople. The order of the switch statements is relevant.